packages/create/templates/app-graphql/tests/graphql.ts

```diff
@@ -1,13 +1,14 @@
-import {buildSchema} from 'graphql';
 import {
   TestGraphQL,
+  createGraphQLSchema,
   createGraphQLFiller,
   createGraphQLController,
   type GraphQLController,
 } from '@quilted/quilt/graphql/testing';
 
-import schema from '../graphql/schema.ts';
+import schemaSource from '../graphql/schema.ts';
```

Comment:
It would be nice if this was done at build time instead, with types alongside the schema files, like for documents

Reply:
https://github.com/lemonmade/quilt/issues/574